packages/testing/register-chai-plugins.js

```diff
@@ -2,7 +2,7 @@
 import { chaiDomDiff } from '@open-wc/semantic-dom-diff';
 import { chaiA11yAxe } from 'chai-a11y-axe';
 
-import chai from '@esm-bundle/chai';
+import * as chai from 'chai';
```

Comment:
Running into https://github.com/chaijs/chai/issues/1603 when trying to register all the plugins further down in this file. It needs to be fixed before we can land this PR.

Reply:
We can simple use `import { use } from 'chai'` and change `chai.use()` by `use()`